### PR TITLE
Update line2route and shorten README

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: R/od-funs.R [hunk 1]
```diff
@@ -139,27 +139,29 @@ line2points <- function(l){
...
-    tryCatch({
+  if(class(ldf) == "SpatialLinesDataFrame") ldf <- line2df(ldf)
 
-      rfnew <- route_cyclestreet(from = ldf[i, 1:2], to = ldf[i,
-                                                              3:4], ...)
-      row.names(rfnew) <- as.character(i)
+  rf <- route_cyclestreet(from = ldf[1,1:2], to = ldf[1, 3:4])
+  tryCatch({
+    rf <- route_cyclestreet(from = ldf[i,1:2], to = ldf[i, 3:4], ...)
+  }, error = function(e){print(paste0("Fail for line number ", 1))})
```
Comment (richardellison) [inline R/od-funs.R]: Probably best to use warning() here instead of print().

File: R/od-funs.R [hunk 2]
```diff
@@ -139,27 +139,29 @@ line2points <- function(l){
...
       if(!exists("rf")){
-        rf <- rfnew
-      } else {
+        rf <- route_cyclestreet(from = ldf[i,1:2], to = ldf[i, 3:4], ...)
+      }else{
+        rfnew <- route_cyclestreet(from = ldf[i,1:2], to = ldf[i, 3:4], ...)
+        row.names(rfnew) <- as.character(i)
         rf <- maptools::spRbind(rf, rfnew)
       }
+    }, error = function(e){print(paste0("Fail for line number ", i))})
```
Comment (richardellison) [inline R/od-funs.R]: Same thing here (warning() instead of print()).

File: R/od-funs.R [hunk 3]
```diff
@@ -139,27 +139,29 @@ line2points <- function(l){
...
-      print(paste0("Fail for line number ", i))
-    })
-    perc_temp <- i%%round(nrow(ldf)/10)
-    if (!is.na(perc_temp) & perc_temp == 0) {
-      print(paste0(round(100 * i/nrow(ldf)), " % out of ",
-                   nrow(ldf), " distances calculated"))
+    # Status bar
+    perc_temp <- i %% round(nrow(ldf) / 10)
+    if(!is.na(perc_temp) & perc_temp == 0){
+      print(paste0(round(100 * i/nrow(ldf)), " % out of ", nrow(ldf),
```
Comment (richardellison) [inline R/od-funs.R]: message() instead of print() here I think would be best.

**Comment by Robinlovelace** [comment]
Updated now - cheers

**Pull request merged**